mimic/GUI/DebugLogWindow.py

```diff
@@ -23,6 +24,12 @@ def __init__(self, master: tk.Tk):
         super().__init__(master)
 
         self.master = master
+
+        # TkInter photoimage only supports PPM PGM image formats
+        # Using ImageTk and Image from PIL gets around this
+        mimic_logo = ImageTk.PhotoImage(Image.open("./assets/favicon.ico"))
```

Comment:
Create icon after hiding the window to prevent windows from appearing during startup